utils.js

```diff
@@ -43,4 +43,53 @@ function _in(needle, haystack) {
     return false;
 }
 
-module.exports = {parsePrice, composeRanges, _in};
\ No newline at end of file
+function formatHeader(player, auctionMode) {
+
+    let arrayBuffer = [];
+    let padding = ''
+    //70 for iphone, 124 for desktop
```

Comment:
Wait, so this is not dynamic (not sure if we CAN do stuff that's dynamic when it's just for a message) but you've just chosen to optimize for mobile and leave it at that? I guess I'm fine with this, just curious since you put the note for the desktop value here.

Reply:
Man, sorry- I had a lot of old code I didn't realize that I should have deleted before pushing.  This was an experiment.  I was trying to write a function that would be like a 'float right' with padding spaces before the word, but because it's not a mono-spaced font it doesn't work well.  Can ignore- will delete.